tests/e2e.sh

```diff
@@ -17,7 +17,7 @@
 set -e
 set -o pipefail
 
-KUBERNETES_VERSION=v1.17.3
+KUBERNETES_VERSION=v1.18.0
```

Comment:
There was a newer k8s version do you mind bumping to the latest here, thanks!

Reply:
Yes, sure. v1.18.1? the other releases are in alpha or beta stage.

Reply:
Yes that is that what I meant.